routemaster/state_machine/gates.py

```diff
@@ -14,8 +15,16 @@
 )
 from routemaster.state_machine.exceptions import DeletedLabel
 
+logger = logging.getLogger(__name__)
```

Comment:
This logger appears unused -- is that intentional?

Reply:
Ah yep we can remove this now. d2f667106e6d0b2ff7d1ba0a240f397df64fd2f1